util/fipstools/inject_hash/macho_parser/macho_parser.h

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#ifndef MACHO_PARSER_H
+#define MACHO_PARSER_H
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <mach-o/loader.h>
+#include <mach-o/nlist.h>
+
+typedef struct {
+    char name[16];
+    size_t size;
+    uint32_t offset;
+} section_info;
+
+// Since we only support 64-bit architectures on Apple, we don't need to account for any of the 32-bit structures
+#define LC_SEG LC_SEGMENT_64
+#define BIT_MODIFIER 8
```

Comment:
Something is counting in bytes and other things in bits and you need to go from bits to bytes somewhere?

Reply:
* Based on the description, it seems the name of this macro/constant would more appropriately be `BYTES_PER_WORD`?

Reply:
Taking another look at this, I believe I can remove the need for a macro at all